### DISCUSSION TITLE
예제 실행방법문의 demo. opensdk 에서

### DISCUSSION THREAD

**Discussion by dogfootman** [fsoftwareengineer/todo-application#16] [질문과 답변] [answered]
mac. bicsur 에서 clone 받아서 실행해 봤는데 안되고 있어서 문의드립니다.

openjdk version "17.0.1" 2021-10-19
OpenJDK Runtime Environment (build 17.0.1+12-39)
OpenJDK 64-Bit Server VM (build 17.0.1+12-39, mixed mode, sharing)

IntelliJ IDEA 와 Eclipse 에서 모두 실행이 되지 않습니다.

demo 실행 : todo-application/6.3-Backend_Deployment ...  에서 
./gradlew bootRun 을 하면 버전이 맞지 않는지 실행이 되지 않습니다.

Could not compile settings file '/Users/noman/IdeaProjects/todo-application/6.3-Backend_Deployment_With_AWS_ElasticBeanstalk/demo/settings.gradle'.
> startup failed:
  General error during conversion: Unsupported class file major version 61
  
  java.lang.IllegalArgumentException: Unsupported class file major version 61

**Comment by fsoftwareengineer** [comment, marked as answer]
JDK 버전을 1.8.0_241 또는 opendjk 1.8로 변경 해 실행 해 주시기 바랍니다.
